patches/node/fix_allow_preventing_initializeinspector_in_env.patch

```diff
@@ -1,18 +1,17 @@
 From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: Shelley Vohr <shelley.vohr@gmail.com>
-Date: Tue, 1 Sep 2020 19:30:08 -0700
+Date: Tue, 22 Sep 2020 19:44:30 -0700
 Subject: fix: allow preventing InitializeInspector in env
 
 https://github.com/nodejs/node/commit/8c5ad1392f30cfe6b107e9bd85f4cb918ba04aab
 made it such that env->InitializeInspector was called in CreateEnvironment
-no matter what, which creates an issue for embedders seeking to manage
-the InspectorAgent themselves as Electron does. This adds a new
+no matter what, which creates an issue for Electron, as the V8 inspector
+already exists in the renderer process and therefore we only want to
+initialize it in the browser process. This adds a new
 EnvironmentFlags option which allows preventing that invocation.
 
-This will be upstreamed.
```

Comment:
are we no longer intending to upstream this?

Reply:
oh we are, i have a PR opened already! my bad